.github/actions/configure-credentials/action.yaml

```diff
@@ -0,0 +1,20 @@
+name: "Configure Credentials action"
+description: "Wrapper for the configure AWS credentials action"
+
+inputs:
+  secret_account_id:
```

Comment:
change to aws_account_id ?  The account id is a secret but that is passed in from github env ; just not sure secret here helps understand that it is an aws account id

Reply:
Done